Scripts/assignment/emme_assignment.py

```diff
@@ -425,18 +425,27 @@ def calc_noise(self):
                 speed = (0.3*(60*link.length/link["@car_time_aht"])
                          + 0.7*link.data2)
             speed = max(speed, 50.0)
-
+            
             # Calculate start noise
-            if speed <= 90:
-                heavy_correction = (10*log10((1-heavy_share)
-                                    + 500*heavy_share/speed))
+            log.info("NEW FORMULA FOR NOISE CALC")
+            if speed >= 50:
+                LAE_light = 73.5 + 25*log10(speed/50) 
+                LAE_heavy = 80.5 + 30*log10(speed/50)
+            if speed >= 40:
```

Comment:
:exclamation: This is not an `elif` clause, so it will always override the previous `if`.